### PR TITLE
Add WebNN Core ML implementation status

### PR DESCRIPTION
**Pull request by ibelem**
To solve the issue https://github.com/webmachinelearning/webmachinelearning.github.io/issues/70 reported by @oscarbg , added latest WebNN Core ML implementation status.

@philloooo @huningxin @anssiko PTAL.

coreml

### CODE REVIEW

**Review by anssiko — APPROVED** [review]
LGTM!

@philloooo @huningxin to cross-check Chromium milestones.

File: assets/json/webnn_status.json [hunk 1]
```diff
@@ -1374,9 +1374,9 @@
       ],
       "dml_progress": 4,
       "dml_chromium_version_added": "M122",
-      "coreml_op": [""],
-      "coreml_progress": 2,
-      "coreml_chromium_version_added": "",
+      "coreml_op": ["linear"],
```
Comment (philloooo) [inline assets/json/webnn_status.json]: this is done with ` add(mul(alpha, a), beta)` instead.

File: assets/json/webnn_status.json [hunk 2]
```diff
@@ -16,9 +16,9 @@
       ],
       "dml_progress": 4,
       "dml_chromium_version_added": "M122",
-      "coreml_op": [""],
-      "coreml_progress": 2,
-      "coreml_chromium_version_added": "",
+      "coreml_op": ["reduce_argmax"],
```
Comment (philloooo) [inline assets/json/webnn_status.json:19]: this is not implemented yet

**Review by philloooo — COMMENTED** [review]
Thanks for adding the CoreML report!
Aside from the comments, `Neg` is also supported

File: assets/json/webnn_status.json (hunk 2 above)
Comment (ibelem) [reply assets/json/webnn_status.json:19]: Removed version_added for argmax and argmin and set the progress to WIP. Fixed.

File: assets/json/webnn_status.json (hunk 1 above)
Comment (ibelem) [reply assets/json/webnn_status.json]: Thanks, fixed.

**Comment by ibelem** [comment]
Thanks @philloooo for the review feedback, added Neg status and fixed the other issues raised. PTAL.

**Comment by anssiko** [comment]
@philloooo thanks for reviewing the Core ML implementation status.

@huningxin @philloooo I'll merge this now. Any further changes or updates, please propose and @ibelem will create a new PR for your review.

**Pull request merged**